src/app/static/src/app/components/plots/bar/utils.ts

```diff
@@ -154,10 +155,10 @@ export const getErrorLineAnnotations = function(chartData: BarChartData,
         if (!dataset) {
             return
         }
-        const errorBarData = dataset.errorBars;
-        const label = dataset.label;
+        const errorBarData = dataset.errorBars!!;
+        const label = dataset.label || "";
```

Comment:
`dataset.label` has type `string | undefined` so providing default value for error bar label in case `undefined`